### PR TITLE
Done

### PR DESCRIPTION
**Pull request by Terry-Thompson**

### CODE REVIEW

**Comment by bhollan** [comment]
Hi @Terry-Thompson !

Looking over your changes here, it looks like you have an extra opening `div` tag in your card in `index.html`.  Otherwise, I think we can merge your change in!

As always, feel free to Ask A Question if you need any help getting things squared away.
CC: @aturkewi

**Comment by Terry-Thompson** [comment]
Awesome.  Thank you sir!

File: index.html [hunk 1]
```diff
@@ -25,6 +25,18 @@
         </div>
 
 
+          <div class="roster-cards-container">
```
Comment (aturkewi) [inline index.html]: Specifically, this line should be removed, and then you should move your entire `student-card` back one tab. 
(You can do this this in the Learn IDE by highlight all the text and then doing `cmd  + [/]` on mac or `ctrl + [/]` on windows. `[` to move indent left, or `]` to move indent right.

**Comment by aturkewi** [comment]
Hello! We recently made a change to this repo that caused all open PRs to be close and we are not able to re-open them. I'm very sorry about the inconvenience, but I just wanted to let you know why this PR was close and that we still welcome your profile on the site! So if you're still interested in having your profile up, please feel free to open another PR.

**Pull request closed without merging**